### ISSUE TITLE
Unable to install biegunka via nix because of failing tests

### ISSUE THREAD

**Issue by dmalikov** [biegunka/biegunka#62]
```
1) Control.Biegunka.Biegunka.expandHome expands ~$user to the $user user home directory
expected: "/homeless-shelter"
 but got: "/var/empty"

2) Control.Biegunka.Biegunka.expandHome expands ~$user/$path to $path in the $user user home directory
expected: "/homeless-shelter/foo"
 but got: "/var/empty/foo"
```

Full log: http://nixpaste.noip.me/ZhKfSVbC

**Comment by dmalikov** [comment]
Issue was not about biegunka at all

**Comment by supki** [comment]
I'm hitting these errors too with `nix-build`. Are you sure biegunka isn't responsible?

**Comment by supki** [comment]
Fixed in 6d3b26b28ef6d30e81df57d62c0720a0c107fb7d
